examples/simple-mirror.py

```diff
@@ -0,0 +1,63 @@
+import matplotlib.pyplot as plt
+import numpy as np
+
+import traceon.geometry as G
+import traceon.solver as S
+import traceon.excitation as E
+import traceon.plotting as P
+import traceon.tracing as T
+from traceon.interpolation import FieldRadialAxial
+
+
+boundary = G.Path.line([0, 0, -1], [2, 0, -1]).line_to([2, 0, 1]).line_to([0.3, 0., 1])
+mirror = G.Path.line([0., 0., 0.], [1., 0., 0.])
+
+boundary.name = 'boundary'
+mirror.name = 'mirror'
+
+mesh = (boundary + mirror).mesh(mesh_size_factor=45)
+
+# Show the generated mesh, with the given electrode colors.
+P.plot_mesh(mesh, ground='green', lens='blue', show_normals=True)
```

Comment:
Please change names to correspond with 'boundary', 'mirror'

Reply:
There is also no point in showing normals if you are using only voltage excitations

Reply:
fixed. see new commit